0_intro_to_ml_using_jax.ipynb

```diff
@@ -0,0 +1,3758 @@
+{
```

Comment:
<p>Shall we all come up with some possible answers to these questions so that we don't have to think about it during the prac?</p><p> </p><ol><li>Model complexity (too high or too low resulting in over- or under-fitting), Poor optimisation (too high or low learning rate, noisy optimisation landscape, etc)</li><li>Collect more data, make sure your data covers the relevant domain well, add some regularisation techniques to your optimisation process (Weight decay, dropout, etc)</li><li>we assume the seen and unseed data are distributed similarly (look roughly the same): they cover the same domain, data spread over the domain is similar, the data is spread over the domain in the same way</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='201'/>

Reply:
<p>Let's add this the prac channel so that we have these great discussion points! </p>